plenum/server/pool_manager.py

```diff
@@ -252,10 +253,16 @@ def getNodeName(self, nym):
         return nodeTxn[DATA][ALIAS]
 
     def checkValidOperation(self, operation):
-        checks = []
+        # data exists and is dict
         if operation[TXN_TYPE] == NODE:
-            checks.append(DATA in operation and isinstance(operation[DATA], dict))
-        return all(checks)
+            if DATA not in operation:
+                return "'{}' is missed".format(DATA)
+            if not isinstance(operation[DATA], dict):
+                return "'{}' is not a dict".format(DATA)
+
+        # VerKey must be base58
+        if len(set(operation[TARGET_NYM]) - set(base58.alphabet)) != 0:
```

Comment:
Agree. As I know we are going to do some validation story in the future. We can implement verkey validation in that future story. In order to close the ticket that fix fits. I hope.